--- origsrc/sugar-0.98.8/bin/sugar-session	2013-05-22 09:26:45.000000000 -0500
+++ src/sugar-0.98.8/bin/sugar-session	2013-10-21 22:46:48.634491600 -0500
@@ -52,9 +52,9 @@ try:
 except ImportError:
     logging.debug('Could not load xklavier for keyboard configuration')
 
-GLib.threads_init()
-Gdk.threads_init()
-dbus.glib.threads_init()
+#GLib.threads_init()
+#Gdk.threads_init()
+#dbus.glib.threads_init()
 
 Gst.init(sys.argv)
 
